preliz/distributions/betabinomial.py

```diff
@@ -83,7 +79,7 @@ def pdf(self, x):
         return np.exp(self.logpdf(x))
 
     def cdf(self, x):
-        if isinstance(x, (np.ndarray, list, tuple)):
+        if isinstance(x, np.ndarray | list | tuple):
```

Comment:
Is this change requiered by ruff?

Reply:
Yup, this was suggested by ruff UP038

Reply:
Do you think we should go ahead with correcting the UP038, or just ignoring it?

Reply:
Let's keep the union type.